tools/site-verification-group-sync/sync-owners.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+syncOwners.py reads the membership of a Google Group and
+uses that to populate the "verified owners" of a domain
+in the Google Search Console
+'''
+
+import httplib2
+import json
+
+from apiclient.discovery import build
+from oauth2client.client import OAuth2WebServerFlow
+from google.oauth2 import service_account
+
+def get_config():
+    '''
+    Read config from json file
+    '''
```

Comment:
After this documentation you don't have an empty line but in subsequent ones you do. Pick one :)